### ISSUE TITLE
Skybox Glitching out at very deep Y levels

### ISSUE THREAD

**Issue by Quizer9O8** [Barteks2x/CubicChunks#167]
![2017-02-12_10 44 48](https://cloud.githubusercontent.com/assets/19509275/22861101/d801c27e-f110-11e6-8a3d-79f0a6a29bf2.png)
![2017-02-12_10 47 40](https://cloud.githubusercontent.com/assets/19509275/22861102/d801f2c6-f110-11e6-8bf6-8b0a92df4369.png)

**Comment by Niko-sk2x** [comment]
This is a known bug, I know vanilla does something weird with skybox rendering (I think something is always rendered at y=0 and then causes z-fighting). This is very low priority as I don't think anyone will ever legitimately get that deep.

**Comment by Quizer9O8** [comment]
I might have found the cause of this problem. Whenever you enter the void at Y:-5 you will enter a black square and if you look up you'll see the skybox above. The deeper you go the smaller the square gets.
![2017-02-16_19 43 37](https://cloud.githubusercontent.com/assets/19509275/23037363/5f584b44-f486-11e6-9895-39d6b79528d0.png)
![2017-02-16_19 43 27](https://cloud.githubusercontent.com/assets/19509275/23037364/5f58ae04-f486-11e6-9271-8c03f39869db.png)

**Comment by Niko-sk2x** [comment]
Yes, this is the cause of the problem (most likely). When the square is too far away, it causes z-fighting with the skybox above it. I'm not planning to fix it anytime soon.

**Comment by Kevadroz** [comment]
Why not just remoing the skybox or moving it to void level?

**Comment by Niko-sk2x** [comment]
It's not a problem of what do do with it. It's a problem of where in the Minecraft code it is and someone taking the time to do it. As it is, the glitching very deep is very minor and unlikely to be done anytime soon. The blackness issue is partially gamebreaking so it may be fixed before alpha.

**Comment by muzikbike** [comment]
From my testing this no longer applies to vanilla 1.15/16, so fixing this might no longer be neccessary if 1.12 support doesn't continue
